webdriver-spec.html

```diff
@@ -3802,7 +3802,7 @@
 <ol>
  <li><p>Let <var>rectangle</var> be
   the first element of the <a><code>DOMRect</code></a> sequence
-  returned by calling <a>getClientRects</a> on <var><a>element</a></var>.
+  returned by calling <a>getClientRects</a> on the <a>element</a>.
```

Comment:
s/the <a>element</a>/<var>element</var>/

We need to mark the `element` in the text just above this as a variable too.